SpinCore_pp/pulse_length_conv.py

```diff
@@ -3,26 +3,27 @@
 import SpinCore_pp as spc
 
 
-def prog_plen(desired_beta, amplitude, deblank_us):
+def prog_plen(desired_beta, settings):
     """
-    Takes the desired β (μs√(W)), amplitude and deblank_us settings
+    Takes the desired β (μs√(W)) and configuration file 
     to calculate what pulse length should be programmed in order to
     get the desired β based on the amplitude and deblank setting.
 
     Parameters
     ==========
-    desired_beta: float
-                    the desired β you wish the spincore to output,
-                    in μs*sqrt(W)
-    amplitude: float
-                amplitude setting of the output rf pulse
-    deblank_us: float
-                deblanking period before the actual output
-                rf pulse
+    desired_beta : float
+        the desired β you wish the spincore to output,
+        in μs*sqrt(W)
+    settings :  dict-like
+        contains the following keys.  It's crucial that these get used in the pulse sequence.
+
+        :amplitude: float
+        :deblank_us: float
+
```

Comment:
see [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html)

Reply:
went through other docstrings in this PR and edited to correspond to that outline